OpenOversight/app/models/database.py

```diff
@@ -39,22 +39,26 @@
 class Department(BaseModel):
     __tablename__ = "departments"
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(255), index=True, unique=True, nullable=False)
+    name = db.Column(db.String(255), index=False, unique=False, nullable=False)
     short_name = db.Column(db.String(100), unique=False, nullable=False)
+    state = db.Column(db.String(2), default="")
```

Comment:
Same comment as in the other PR: We should also decide on whether the default is the empty string (and then make the column non-null) or leave null as the default (combining both should be avoided). Either works for me.
Between default and server_default I think server_default should be preferred. So I think it should be either `db.Column(db.String(2))` or `db.Column(db.String(2), server_default="", nullable=False)`. Let me know if you think I am wrong here, definitely a possibility

Reply:
`server_default` is definitely the right choice here. There are still some modifications that I'm making to this. `default` does not seem to do much here.

Reply:
Done!